core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -602,6 +602,20 @@ secret_configs:
       - cluster: api.ci
         namespace: ci
         name: registry-push-credentials-quay.io-open-cluster-management
+  - from:
+      config.json:
+        dockerconfigJSON:
+        - bw_item: quay.io/openshift-ocs-dev
+          registry_url: quay.io
+          auth_bw_attachment: push-token
+        - bw_item: build_farm
+          registry_url: registry.ci.openshift.org
+          auth_bw_attachment: token_image-puller_app.ci_reg_auth_value.txt
+    to:
+      - cluster: app.ci
+        namespace: ci
+        name: registry-push-credentials-quay.io-ocs-dev
+        type: kubernetes.io/dockerconfigjson
```

Comment:
This `secret/ocs-operator-quay-secret` was generated by the user and mirrored to ci namespace.
We keep the consistency of secrets here.

```
oc --context api.ci get secret -n ci ${secret} -o yaml | yq -r '.data."config.json"' |base64 -d | jq -r '.auths | keys[]'
quay.io
registry.svc.ci.openshift.org
```

The value for quay.io is copied over.
```
$ secret=ocs-operator-quay-secret
$ oc --context api.ci get secret -n ci ${secret} -o yaml | yq -r '.data."config.json"' |base64 -d | jq -r '.auths."quay.io".auth' > ./push-token
```

The value for registry.svc.ci.openshift.org is for accessing the source.
Here, we use the new registry instead since the src image is there.